src/scss/footer.scss

```diff
@@ -0,0 +1,96 @@
+#footer {
+	display: block;
+	min-height: 0 !important;
+	color: #ffffff;
+}
+
+.newsletter {
+	width: 100%;
+	background-color: #222222;
+	padding-top: 100px;
+	padding-bottom: 100px;
+	text-align: center;
+
+	p {
+		text-transform: uppercase;
```

Comment:
If I remember correctly, you should have to redefine most of these text-related properties. They are all defined in the _typography.scss file. The only thing you might want to add is the ```text-transform: uppercase;``` which normally doesn't apply to ```p``` in the rules set in the typography file.